Sources/StreamChat/Utils/LazyCachedMapCollection.swift

```diff
@@ -71,7 +82,11 @@ public struct LazyCachedMapCollection<Element>: RandomAccessCollection {
         }
     }
 
-    public func append(_ element: Element) {
+    public mutating func append(_ element: Element) {
```

Comment:
In theory this is a breaking change, but I'm 100% sure nobody is using this, if they do, it is a bug, so they need to be notified that appending a new element, is a mutating func and it will create a new copy.